rfc/rfc009-verifiable-presentations.md

```diff
@@ -92,308 +34,21 @@ The terminology used is defined in https://www.w3.org/TR/did-core/#terminology a
-    "iat":"<issuance date>",
-    "thread_id": "<same identifier as used in inbound message>",
-    "body": {
-        "message": "<error message as specified in use case>"
-    }
-},
-"signature": ""
-
-```
-
-  Date and time values MUST be a number containing a NumericDate value (Unix epoch timestamps).
-
-
-
-## 5. Realization of the verifiable presentation
-
-The verifiable presentation holds the credentials that are presented to the verifier.
-
-```json
-"protected": {
-    "alg": "EdDSA", 
-    "kid": "<reference to assertion method in did document>",
-    "typ": "JWT"
-},
-"payload": {
-    "jti": "<presentation id>",
-    "iss": "<did of holder>",
- 		"aud": "<did of intended verifier>",
- 		"nbf": "<valid from date, same as issuance date or later>",
- 		"exp": "<expiration date>",
- 		"iat": "<issuance date>",
-    "vp": {
-        "@context": ["https://www.w3.org/2018/credentials/v1"], 
-        "type": ["VerifiablePresentation"], 
-        "verifiableCredential": ["<base64url-encoded JWT as string>"]
-    }
-},
-"signature": "<signature of the holder>"
-
-```
-
-Date and time values MUST be a number containing a NumericDate value (Unix epoch timestamps).
-
-
-
-## 6. Realization of the verifiable credential
-
-The JWT of the credential.
-
-```json
-"protected": {
-    "alg": "EdDSA", 
-    "kid": "<reference to assertion method in did document>",
-    "typ": "JWT"
-},
-"payload": {
-    "jti": "<credential id>",
-    "iss": "<did of issuer>",
- 		"sub": "<did of holder>",
-    "nbf": "<valid from date, same as issuance date or later>",
-    "exp": "<expiration date>",
-    "iat": "<issuance date>",
- 		"vc": {
-        <json of the verifiable credential>
-    }
-},
-"signature":
-```
-
-Date and time values MUST be a number containing a NumericDate value (Unix epoch timestamps).
-
-
-
-## 7. Realization of the status registry
-
-The verifiable credential SHOULD have a property for the discovery of information about the current status of a verifiable credential. The property is specified in the [Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/#status).
-
-Allowed status of a verifiable credential is valid, suspended, or revoked.
-
-```json
-  "credentialStatus": {
-    "id": "https://example.com/status/<status list identifier>",
-    "type": "CredentialStatusList2017"
-  }
-```
-
-The credentials list is specified by [Credential Status List (2017)](https://w3c-ccg.github.io/vc-csl2017/).
+## 3. Requirements for verifiable credentials
 
+The Nuts framework MUST support the use of verifiable credentials in the roles of Issuer, Holder, and Verifier as described in [Verifiable Credentials Use Cases](https://www.w3.org/TR/vc-use-cases/) and [W3C Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/). In the terms of decentralized Identifiers, the Nuts framework acts as a software agent of the person or organisation accountable for the role of Issuer, Holder, and Verifier.
```

Comment:
Holder and Verifier sounds logical. I think the Issuer part is optional. Not every node/implementation MUST be able to issue VCs. Unless I'm missing a use-case?

Reply:
The issuer part is optional now. I think its a priority issue.